recipes/pyddx/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "pyddx" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d5cde4473c19b836a77948f79e33ec15b8040fdbf2278a8f02d7c81df4f09111" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ddsolvation/ddx/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win or py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - make
+    - cmake >=3.14
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - pip
+    - pybind11 >=2.6
+    - setuptools
+    - libblas
+    - liblapack
+  run:
+    - python
+    - numpy >=1.14
+
+test:
+  imports:
+    - pyddx
+  requires:
+    - pytest
+  commands:
+    - export PYLIB_EXT=`$PYTHON -c 'import sysconfig as s; print(s.get_config_var("EXT_SUFFIX"))'`
+    - test -f $PREFIX/lib/python{{ PY_VER }}/site-packages/pyddx$PYLIB_EXT
```

The import is already checked above, please check the metadata here.
```suggestion
  requires:
    - pip
  commands:
    - pip check
```

---

Adding a `pip check` leads to a failing test stage because pyddx currently requires the ninja wheel and pybind11 as dependencies. I'll add it and we can discuss options.

---

Then we should patch their `setup.py` to remove that requirement.

---

So I should patch our ;). I'm removing ninja now, but I'm not so sure how to deal with pybind11. Python dependency management keeps changing all the time and I'm honestly not fully up to speed, so any help would be appreciated: https://github.com/ddsolvation/ddX.

---

Oh I get it. I think I confused install_requires and setup_requires. I'll fix it upstream.

---

Cool. Seems to be working now @xhochy, thanks :+1: